modules/network-observability-configuring-custom-metrics.adoc

```diff
@@ -0,0 +1,201 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
```

Comment:
@jotak originally, I had this procedure right after [Custom metrics ](https://github.com/openshift/openshift-docs/pull/74412/files#diff-0cb28d92ab17f03e9453aa0edd11dbd5d65d9017e0d4614c9e1d2a7ba42a3345)in the _Using metrics with dashboards and alerts_ section. I'm not 100% sold on it being in Configuring, but I tried moving it there and creating Additional resource links to see. It has a FlowCollector configuration piece, so in a way, it could indeed go in the Operator configuration. 

To give an argument for why we may consider moving it back to Using metrics with dashboards and alerts, many of the procedures in [Observing](https://docs.openshift.com/container-platform/4.12/observability/network_observability/observing-network-traffic.html) network traffic have FlowCollector configurations too, but we keep them in Observing because they tie in more with the web console views and they aren't really required configurations, similar to this one.

Reply:
In fact these steps configuring FlowCollector are not necessary for using FlowMetrics, it's just that they enable new features that come with new fields in flow, and since the metrics API allows to gather data from any flow field, these new fields can be leveraged there. But users may also very well create custom metrics with any of the default/old flow fields

Reply:
Okay, so that gives me a good reason to keep the procedure in the Operator configuration then, since it isn't necessarily only for using FlowMetrics.

Reply:
On second thought, given that the FlowCollector steps aren't necesary any longer, I think I will move it back to the Metrics, alerts and dashboards page.